frontend/lib/models/test_execution.dart

```diff
@@ -127,6 +127,16 @@ enum TestExecutionReviewDecision {
     }
   }
 
+  bool get isDeprecated {
+    switch (this) {
+      case approvedFaultyHardware:
+      case approvedUnstablePhysicalInfra:
+        return true;
```

Comment:
sorry if I don't know how dart works, but shouldn't it look like this:

```suggestion
      case approvedFaultyHardware:
        return true;
      case approvedUnstablePhysicalInfra:
        return true;
```

Reply:
According to the [Dart docs](https://dart.dev/language/branches#switch-statements) the empty case falls through, so in this case the logic for `approvedFaultyHardware` continues through and executes the same case as `approvedUnstablePhysicalInfra`.